Examples/ObjectiveC/BlockingGesturesDelegateExample.m

```diff
@@ -0,0 +1,64 @@
+#import "BlockingGesturesDelegateExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleBlockingGesturesDelegate = @"BlockingGesturesDelegateExample";
+
+@interface BlockingGesturesDelegateExample () <MGLMapViewDelegate>
+@property (nonatomic) MGLCoordinateBounds colorado;
+@end
+
+@implementation BlockingGesturesDelegateExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    mapView.delegate = self;
+    mapView.styleURL = [MGLStyle outdoorsStyleURLWithVersion:9];
+    
+    // Denver, Colorado
+    CLLocationCoordinate2D center = CLLocationCoordinate2DMake(39.748947, -104.995882);
+    
+    // Starting point.
+    [mapView setCenterCoordinate:center zoomLevel:10 direction:0 animated:NO];
+    
+    // Colorado's bounds
+    CLLocationCoordinate2D ne = CLLocationCoordinate2DMake(40.989329, -102.062592);
+    CLLocationCoordinate2D sw = CLLocationCoordinate2DMake(36.986207, -109.049896);
+    self.colorado = MGLCoordinateBoundsMake(sw, ne);
+    
+    [self.view addSubview:mapView];
+}
+
+// This example uses Colorado's boundaries to restrict
+// the camera movement
```

Comment:
The use of a period at the end of a comment is inconsistent in this file. Here there’s no period at the end of the sentence; above, “Starting point” does end with a period, even though it isn’t a sentence. Normally I end all the comments with periods if any of them is a sentence.